apps/modernization-ui/src/apps/search/patient/PatientCriteria/BasicInformation.tsx

```diff
@@ -0,0 +1,195 @@
+import { Control, Controller } from 'react-hook-form';
+import { Selectable, asValue } from 'options';
+import { Input } from 'components/FormInputs/Input';
+import { validNameRule } from 'validation/entry';
+import { DatePickerInput } from 'components/FormInputs/DatePickerInput';
+import { SelectInput } from 'components/FormInputs/SelectInput';
+import { FormGroup, Label, Checkbox, ErrorMessage } from '@trussworks/react-uswds';
+import { RecordStatus } from 'generated/graphql/schema';
+import styles from './basic-information.module.scss';
+import { PatientCriteriaEntry } from '../criteria';
+
+type Props = {
+    control: Control<PatientCriteriaEntry>;
+    handleRecordStatusChange: (
+        value: Selectable[],
+        status: Selectable,
+        isChecked: boolean,
+        onChange: (status: Selectable[]) => void
```

Comment:
Question: Why is this change handler necessary? shouldn't the input just set the form value? If another component is concerned about taking an action on status change, then it should just `useWatch`.